src/NodaTime/LocalDate.cs

```diff
@@ -235,6 +235,7 @@ public static LocalDate FromDateTime(DateTime dateTime)
         /// <returns>A new <see cref="LocalDate"/> with the same values as the specified <c>DateTime</c>.</returns>
         public static LocalDate FromDateTime(DateTime dateTime, CalendarSystem calendar)
         {
+            Preconditions.CheckNotNull(calendar, nameof(calendar));
```

Comment:
(I'm surprised we didn't already have tests that do something like Guava's `NullPointerTester` to exercise all the public static/instance methods.)